.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     macos:
-      xcode: "9.1.0"
+      xcode: "10.2.0"
```

Comment:
Change to 10.2.1
CircleCI are removing support for 10.2.0 in the coming weeks
https://discuss.circleci.com/t/removing-support-for-xcode-9-1-and-10-2-0/31542